src/main/java/Main.java

```diff
@@ -111,7 +111,7 @@ public static void main(String[] args) throws Exception {
 
             _main(args);
         } catch (UnsupportedClassVersionError e) {
-            System.err.println("Jenkins requires Java7 or later, but you are running "+
+            System.err.println("Jenkins requires Java 8 or later, but you are running "+
```

Comment:
Oh I'm sorry, I must be on the wrong branch or repo. I was experiencing this error message in the war for jenkins 2.54+ which now requires java 8. I will try to find the proper place to submit this PR.